File: app/src/main/java/com/app/rupiksha/models/Skey.java
package com.app.rupiksha.models;

import org.simpleframework.xml.Attribute;
import org.simpleframework.xml.Text;
public class Skey {

    public Skey() {
    }

    @Attribute(name = "ci", required = false)
    public String ci;

    @Text(required = true)
    public String value;

}

File: app/src/main/java/com/app/rupiksha/models/Skey.kt
package com.app.rupiksha.models

import org.simpleframework.xml.Attribute
import org.simpleframework.xml.Text

data class Skey(
    @field:Attribute(name = "ci", required = false)
    var ci: String? = null,

    @field:Text(required = true)
    var value: String? = null
)
